app/viewModels/houseConsignment/index/items/additionalReference/AdditionalReferenceTypeViewModel.scala

```diff
@@ -22,26 +22,26 @@ import viewModels.ModeViewModelProvider
 
 import javax.inject.Inject
 
-case class AdditionalReferenceViewModel(heading: String,
-                                        title: String,
-                                        requiredError: String,
-                                        arrivalId: ArrivalId,
-                                        mode: Mode,
-                                        houseConsignmentIndex: Index,
-                                        itemIndex: Index,
-                                        additionalReferenceIndex: Index
+case class AdditionalReferenceTypeViewModel(heading: String,
```

Comment:
What was the decision behind this name change? An additional reference type is a value within an additional reference, so the original name arguably makes more sense